### PR TITLE
Darkmode color fix

### PR DESCRIPTION
**Pull request by uthmanmomen13**

### CODE REVIEW

File: renderer/components/DNav.tsx [hunk 1]
```diff
@@ -161,6 +191,7 @@ const DNavComponent = (props: Props) => {
                 id="update-software-button"
                 icon="cloud-upload-alt"
               />
+
```
Comment (ewc340) [inline renderer/components/DNav.tsx]: Remove line here
```suggestion
```

File: renderer/components/Editor.tsx [hunk 2]
```diff
@@ -74,9 +74,9 @@ interface OwnProps {
   onUploadCode: () => void;
   onUpdateKeyboardBitmap: (keyboardBitmap: number) => void;
   onUpdateKeyboardModeToggle: (isKeyboardToggled: boolean) => void;
-  onInitiateLatencyCheck: () => void;
 }
 
+
```
Comment (ewc340) [inline renderer/components/Editor.tsx]: Remove line here 
```suggestion

```

File: renderer/components/Editor.tsx [hunk 3]
```diff
@@ -239,6 +235,7 @@ export const Editor = (props: Props) => {
 
     props.onUploadCode();
   };
+
```
Comment (ewc340) [inline renderer/components/Editor.tsx]: Remove line 
```suggestion

```

File: renderer/components/Peripheral.tsx [hunk 4]
```diff
@@ -20,6 +20,8 @@ interface OwnProps {
   name: string;
   type: number;
   params: Param[];
+  background: string;
```
Comment (ewc340) [inline renderer/components/Peripheral.tsx]: change to 
```suggestion
  backgroundColor: string;
```

File: renderer/components/Peripheral.tsx [hunk 5]
```diff
@@ -20,6 +20,8 @@ interface OwnProps {
...
+  color: string;
```
Comment (ewc340) [inline renderer/components/Peripheral.tsx]: change to 
```suggestion
  textColor: string;
```

**Review by MeshanKhosla — APPROVED** [review]
LGTM

**Review by ewc340 — CHANGES_REQUESTED** [review]
Just some small changes

File: renderer/components/Peripheral.tsx [hunk 6]
```diff
@@ -32,7 +34,7 @@ export const Peripheral = (props: OwnProps) => {
   const ActualPeripheral = typesToComponents[props.type] || GenericPeripheral;
 
   return (
-    <ListGroupItem style={{ padding: '0px 0px 15px 0px', border: 'none' }}>
+    <ListGroupItem style={{ padding: '0px 0px 15px 0px', border: 'none', backgroundColor: props.background, color: props.color}}>
```
Comment (ewc340) [inline renderer/components/Peripheral.tsx]: Change to
```suggestion
    <ListGroupItem style={{ padding: '0px 0px 15px 0px', border: 'none', backgroundColor: props.backgroundColor, color: props.textColor}}>
```

**Review by ewc340 — APPROVED** [review]
LGTM

**Pull request merged**